assets/src/scripts/monitoring-location/components/hydrograph/selectors/iv-data.js

```diff
@@ -101,7 +101,11 @@ export const getIVDataPoints = memoize(dataKind => createSelector(
             let pointClass;
             const pointQualifiers = point.qualifiers.map(qualifier => qualifier.toLowerCase());
             const maskedQualifier = pointQualifiers.find(qualifier => qualifier in MASKED_QUALIFIERS);
-            const approvalQualifier = pointQualifiers.find(qualifier => qualifier in APPROVAL_QUALIFIERS);
+            // The data services sometimes return more than one approval code per point,
+            // such as ["A", "e"]. In these cases, give priority to 'e', which is 'estimated'.
+            const approvalQualifier = pointQualifiers.includes('e') ? 'e':
+                pointQualifiers.find(qualifier => qualifier in APPROVAL_QUALIFIERS);
```

Comment:
So changing the order of APPROVAL_QUALIFIERS didn't work? I'm not opposed to this but think it could be brittle if more approvals are added (which isn't going to happen).

Reply:
Unfortunately, changing the order didn't work. The qualifiers are in a regular object instead of an array with objects, so it only paid attention to the key names.